apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -331,12 +338,16 @@ defmodule OMG.Watcher.ExitProcessor do
     _ = if not Enum.empty?(events), do: Logger.info("Recognized in-flight exits: #{inspect(events)}")
 
     contract_ife_ids =
-      Enum.map(
-        events,
+      events
+      |> Task.async_stream(
         fn %{call_data: %{in_flight_tx: txbytes}} ->
           ExPlasma.InFlightExit.txbytes_to_id(txbytes)
-        end
+        end,
+        timeout: 50_000,
+        on_timeout: :exit,
+        max_concurrency: System.schedulers_online() * 2
```

Comment:
this would not be larger then our rate limit to infura right (is there a rate limit)? (btw, I actually have no idea what both number are 😅)

Reply:
the beam installs a scheduler per CPU core. so if you have 8 cores times 2, you'd do at most 16 exits at the same time.